data/newMigrations/014-service_entry_provider.js

```diff
@@ -0,0 +1,22 @@
+exports.up = (knex) => {
+    return knex.schema.createTable('service_entry_providers', function (tbl) {
+        tbl.increments('service_entry_provider_id').primary();
+        tbl.int('service_entry_id')
+            .unsigned()
+            .references('services_entry_id')
+            .inTable('service_entries')
+            .onUpdate('RESTRICT')
+            .onDelete('RESTRICT');
+        tbl.int('provider_id')
```

Comment:
Switch the `.int` to `.integer` and add `.notNullable` to service_entry_id and provider_id